src/VolumeDrawable.ts

```diff
@@ -199,43 +199,43 @@ export default class VolumeDrawable {
     // merge to current channel options
     this.channelOptions[channelIndex] = Object.assign(this.channelOptions[channelIndex], options);
 
-    if (Object.hasOwnProperty.call(options, "enabled")) {
+    if (options.enabled !== undefined) {
```

Comment:
I can't remember but I thought we had a good reason for using hasOwnProperty instead of just checking undefined. oh well, this seems ok really.

Reply:
Problem here was that, with all fields of `VolumeChannelDisplayOptions` optional, `hasOwnProperty` didn't guard types properly so that TypeScript knew the fields weren't `undefined`. Looks like `setOptions` just above had already been using that syntax anyways.